lux/lib/lux/signal.ex

```diff
@@ -10,6 +10,7 @@ defmodule Lux.Signal do
           sender: String.t() | nil,
           recipient: String.t() | nil,
           timestamp: DateTime.t(),
+          topic: String.t() | nil,
```

Comment:
Even if early, could we also add a line about this in `lux/guides/signals.livemd`? We can mention that it's for an upcoming feature.